draft-packages/form/KaizenDraft/Form/SearchField/SearchField.tsx

```diff
@@ -40,7 +44,7 @@ export const SearchField = ({
         secondary={secondary}
         {...restProps}
       />
-    </FieldGroup>
+    </div>
```

Comment:
Since this is a `*Field` shouldn't it have validation messages built in? Add a ticket for this.

Reply:
The UI Kit doesn't show it ever having validation messages nor a description. I could add a ticket for design to think about it, but I wouldn't implement it yet if it's not a pattern that's meant to exist.